draft-irtf-cfrg-opaque.md

```diff
@@ -558,46 +653,51 @@ Steps:
 
 ## Envelope Modes {#envelope-modes}
 
-The `EnvelopeMode` specifies the structure and encoding of the corresponding `InnerEnvelope`.
-This document specifies the values of the two aforementioned modes:
+The `EnvelopeMode` specifies the structure and encoding of the
+corresponding `InnerEnvelope`. This document specifies the values
+of the two aforementioned modes:
 
 ~~~
 enum {
- internal(1),
- external(2),
- (255)
+  internal(1),
+  external(2),
+  (255)
 } EnvelopeMode;
 ~~~
```

Comment:
Now that `EnvelopeMode` is no longer encoded anywhere, it seems like we do not need to define this enum anymore, right? An implementation can choose to use whatever number it desires to represent "internal" and "external", it doesn't have to be 1 and 2, right?

Reply:
It doesn't, but let's leave this for a separate PR.